start_production_docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+NODE_ENV=production node dist/apps/doji-backend/main.js &
+cd dist/apps/doji-frontend/
+../../../node_modules/.bin/next start -p 3000 &
+echo "Started Doji Backend and Frontend"
+cd ../../../
+
+nginx
```

Comment:
should this be `nginx -g "deamon off;"`

Reply:
No it should not. We want to daemonize it.

Reply:
ok